kucher/view/main_window/register_view_widget/import_export_dialog.py

```diff
@@ -0,0 +1,314 @@
+#
+# Copyright (C) 2018 Zubax Robotics OU
+#
+# This file is part of Kucher.
+# Kucher is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+# Kucher is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+# of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with Kucher.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+# Author: Pavel Kirienko <pavel.kirienko@zubax.com>
+#
+
+import os
+import yaml
+import asyncio
+import math
+import itertools
+import enum
+
+from logging import getLogger
+from PyQt5.QtWidgets import QWidget, QFileDialog, QTableWidget, QTableWidgetItem, QDialog, QLabel, \
+    QPushButton, QHeaderView, QMessageBox
+from popcop.standard.register import ValueType
+
+from kucher.view.device_model_representation import Register
+from kucher.view.utils import show_error, lay_out_horizontally, lay_out_vertically, get_monospace_font
+
+_logger = getLogger(__name__)
+
+
+class ErrorCodes(enum.Enum):
```

Comment:
Shouldn't say "codes" because it's evident from the semantics. I suggest calling it `CheckResult`.